Provider crashes with null input to nodes

```
swarm_cluster.cluster: Creating...
digitalocean_volume_attachment.storage_worker[1]: Still creating... [10s elapsed]
digitalocean_volume_attachment.storage_worker[0]: Still creating... [10s elapsed]
digitalocean_volume_attachment.storage_worker[2]: Still creating... [10s elapsed]
digitalocean_volume_attachment.storage_worker[0]: Creation complete after 13s [id=275988176-13d8ee7a-526d-11ec-9d60-0a58ac14a139-20211201060907682200000001]
digitalocean_volume_attachment.storage_worker[1]: Creation complete after 13s [id=275988180-13c2a640-526d-11ec-9d60-0a58ac14a139-20211201060907781100000002]
digitalocean_volume_attachment.storage_worker[2]: Creation complete after 13s [id=275988173-13bcc812-526d-11ec-91b0-0a58ac14a2a4-20211201060908392500000003]
╷
│ Error: Plugin did not respond
│
│   with swarm_cluster.cluster,
│   on main.tf line 82, in resource "swarm_cluster" "cluster":
│   82: resource "swarm_cluster" "cluster" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ApplyResourceChange call. The plugin logs may contain more details.
╵

Stack trace from the terraform-provider-swarm_v1.1.0 plugin:

panic: runtime error: index out of range [0] with length 0

goroutine 40 [running]:
github.com/aucloud/terraform-provider-swarm/swarm.resourceClusterCreate({0x18a7928, 0xc00040cfc0}, 0xc00044f778, {0x177f220, 0xc00050a288})
	github.com/aucloud/terraform-provider-swarm/swarm/resource_cluster.go:113 +0xde5
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).create(0xc000138620, {0x18a78b8, 0xc000694280}, 0x2, {0x177f220, 0xc00050a288})
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.9.0/helper/schema/resource.go:341 +0x12e
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).Apply(0xc000138620, {0x18a78b8, 0xc000694280}, 0xc0002d8f70, 0xc00015f000, {0x177f220, 0xc00050a288})
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.9.0/helper/schema/resource.go:467 +0x871
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ApplyResourceChange(0xc00000d590, {0x18a78b8, 0xc000694280}, 0xc0003b2320)
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.9.0/helper/schema/grpc_provider.go:977 +0xd8a
github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ApplyResourceChange(0xc00007d6c0, {0x18a7960, 0xc0004b41e0}, 0x0)
	github.com/hashicorp/terraform-plugin-go@v0.4.0/tfprotov5/tf5server/server.go:332 +0x6c
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ApplyResourceChange_Handler({0x177ae80, 0xc00007d6c0}, {0x18a7960, 0xc0004b41e0}, 0xc00040c660, 0x0)
	github.com/hashicorp/terraform-plugin-go@v0.4.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:380 +0x170
google.golang.org/grpc.(*Server).processUnaryRPC(0xc000280a80, {0x18b4a30, 0xc000428000}, 0xc000432000, 0xc0002f0060, 0x1d6d6c0, 0x0)
	google.golang.org/grpc@v1.42.0/server.go:1282 +0xccf
google.golang.org/grpc.(*Server).handleStream(0xc000280a80, {0x18b4a30, 0xc000428000}, 0xc000432000, 0x0)
	google.golang.org/grpc@v1.42.0/server.go:1616 +0xa2a
google.golang.org/grpc.(*Server).serveStreams.func1.2()
	google.golang.org/grpc@v1.42.0/server.go:921 +0x98
created by google.golang.org/grpc.(*Server).serveStreams.func1
	google.golang.org/grpc@v1.42.0/server.go:919 +0x294

Error: The terraform-provider-swarm_v1.1.0 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.
```

--- a/swarm/resource_cluster.go
+++ b/swarm/resource_cluster.go
@@ -109,6 +109,15 @@ func resourceClusterCreate(ctx context.Context, d *schema.ResourceData, m interf
 	}
 
 	managers := vmnodes.FilterByTag(swarm.RoleTag, swarm.ManagerRole)
+	if len(managers) != 1 {
+		diags = append(diags, diag.Diagnostic{
+			Severity: diag.Error,
+			Summary:  "No managers found in cluster config",
+			Detail:   "At least one manager must exist in the cluster config! Please check your `nodes` configuration.",
+		})
+		return diags
+	}
+
 	if swarmManager.Runner() == nil {
 		if err := swarmManager.SwitchNode(managers[0].PublicAddress); err != nil {
 			diags = append(diags, diag.Diagnostic{

